### ISSUE TITLE
Hellblade: Senua's Sacrifice VR Edition (747350)

### ISSUE THREAD

**Issue by mimattr** [ValveSoftware/Proton#4370]
# Whitelist Request
- Name of the game to be whitelisted: **Hellblade: Senua's Sacrifice VR Edition**
- Steam AppID of the game: **747350**

## System Information
- GPU: **RTX 2060**
- Driver/LLVM version: **NVIDIA 455.38**
- Distro version: **Ubuntu 20.04 LTS**
- [Link to full system information report as Gist](https://gist.github.com/mimattr/b0b7f46366e31e254460b9593ce87e51)
- Proton version: **5.13-2 RC3**
- SteamVR version: **1.15.9**

## I confirm:
- [x] that pressing the `Play` button in the Steam client is sufficient.

## Issues
- [x] I haven't experienced any issues.
- [x] There are no issues left open for this game.

**Comment by gareth-anthony-hulse** [comment]
Doesn't work for me. I just get an error message:
 ![Screenshot_20230318_113223](https://user-images.githubusercontent.com/107568353/226103018-707d17e6-6c3a-4627-ad22-4d268f24ac50.png)

Adding `-d3d12` to launch options results to this:
![Screenshot_20230318_113658](https://user-images.githubusercontent.com/107568353/226103272-dd4cce75-49d9-490f-8e49-fee06ff74153.png)

Error message is hard to read. Here it is zoomed in:
![Screenshot_20230318_113913](https://user-images.githubusercontent.com/107568353/226103363-f6da1102-71c4-458a-b55e-68dab6a91938.png)

My specs: https://linux-hardware.org/?probe=06d89bd973
SteamVR version: 1.25.6

**Comment by kisak-valve** [comment]
Hello @mimattr, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.)

**Comment by gareth-anthony-hulse** [comment]
@kisak-valve Did you mean me, not @mimattr? Anyway, here is the log: [steam-747350.log](https://github.com/ValveSoftware/Proton/files/11010300/steam-747350.log)

**Comment by kisak-valve** [comment]
Indeed, tab complete failed me while your username was scrolled off screen.

From the log:

`Options: {'enablenvapi', 'forcelgadd'}`
`warn:seh:OutputDebugStringA "LowLevelFatalError [File:F:\\P4Workspaces\\Hellblade\\VRTest\\UE4\\Engine\\Source\\Runtime\\Windows\\D3D11RHI\\Private\\D3D11Util.cpp] [Line: 227] \nNvAPI_D3D11_MultiGPU_GetCaps(&MultiGPUCaps) failed \n at F:\\P4Workspaces\\Hellblade\\VRTest\\UE4\\Engine\\Source\\Runtime\\Windows\\D3D11RHI\\"...`
This looks like dxvk-nvapi was enabled (manually?) and is having a hard time. It might be interesting to test how the game behaves with the game's launch options set to `PROTON_ENABLE_NVAPI=0 %command%`.

**Comment by gareth-anthony-hulse** [comment]
I added `PROTON_ENABLE_NVAPI=1` to `/etc/environment` to avoid adding it manually to every game that supports DLSS. Setting launch options to `PROTON_ENABLE_NVAPI=0 %command%` fixed the issue.

**Comment by kisak-valve** [comment]
It might be worthwhile to report your findings to the upstream [dxvk-nvapi](https://github.com/jp7677/dxvk-nvapi/issues/) issue tracker.

**Comment by gareth-anthony-hulse** [comment]
Issue has been fixed: https://github.com/jp7677/dxvk-nvapi/pull/114.
